src/Network/PacketHandlers.cs

```diff
@@ -2538,28 +2538,30 @@ private static void OpenBook(Packet p)
                     BookPageCount = p.ReadUShort(),
                     //title allows only 47 dots (. + \0) so 47 is the right number
                     BookTitle =
-                        new MultiLineBox(new MultiLineEntry(BookGump.DefaultFont, 47, 150, 150, BookGump.IsNewBookD4, FontStyle.None, 0), editable)
+                        new TextBox(new TextEntry(BookGump.DefaultFont, 47, 150, 150, BookGump.IsNewBook, FontStyle.None, 0), editable)
                         {
                             X = 40,
                             Y = 60,
                             Height = 25,
                             Width = 155,
                             IsEditable = editable,
-                            Text = oldpacket ? p.ReadASCII(60).Trim('\0') : p.ReadASCII(p.ReadUShort()).Trim('\0')
+                            Text = oldpacket ? p.ReadUTF8StringSafe(60).Trim('\0') : p.ReadUTF8StringSafe(p.ReadUShort()).Trim('\0')
```

Comment:
the `ReadUTF8StringSafe(X)` should never add `\0`, so probably it's unecessary the Trim('\0')